firefly/kernel/console/console_BACKUP_802957.cpp

```diff
@@ -0,0 +1,111 @@
+#include "firefly/console/console.hpp"
```

Comment:
I think you accidentally uploaded this file. Please avoid using `git add .` in the future.